woocommerce/google-listings-and-ads
# Fix popover width

### Changes proposed in this Pull Request:

Closes #235.

Apparently the popover does not have a default width now, so by default the popovers will be displayed in a very thin vertical rectangle, where its width depends on the longest word inside the popover.

This PR specifies a sensible default width for different view breakpoint in the `HelpPopover` component so that the popover appears more natural to the eyes.

### Screenshots:

Before - the size of the popover depends on the longest word in the content: 

![image](https://user-images.githubusercontent.com/417342/115901843-513d0280-a494-11eb-9997-93a041c2d7b7.png)

After - a more sensible default width:

![image](https://user-images.githubusercontent.com/417342/115901773-32d70700-a494-11eb-836a-21031437d552.png)

### Detailed test instructions:

1. Open any page with a HelpPopover component, e.g. Step 3 in https://gla1.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-mc as per above screenshots.
2. Click on the popover.
3. The popover should display nicely as per the "After" screenshot above.

### Changelog Note:

Fix popover width.
